cypress/page/filling-form.page.ts

```diff
@@ -0,0 +1,133 @@
+import { Hobbies } from "./hobbies-enum.page";
+
+class FillingFormPage {
+    private nameSelector: string
+    private lastNameSelector: string
+    private emailSelector: String
+    private genderSelector: String
+    private dateOfBirthSelector: String
+    private mobileNumberSelector: String
+    private hobbiesSelector: String
+    private currentAddressSelector: String
+    private pageURL: string
+    private submit: string
+    private stateSelector: string
+    private citySelector: string
+
+    constructor(){
+        this.nameSelector = "#firstName"
+        this.lastNameSelector = "#lastName"
+        this.emailSelector = "#userEmail"
+        this.genderSelector = "#genterWrapper .col-md-9"
+        this.dateOfBirthSelector = "#dateOfBirthInput"
+        this.mobileNumberSelector = "#userNumber"
+        this.hobbiesSelector = "#hobbiesWrapper .col-md-9"
+        this.currentAddressSelector = "#currentAddress"
+        this.stateSelector = "#state svg"
+        this.citySelector = "#city svg"
+        this.pageURL = "https://demoqa.com/automation-practice-form"
+        this.submit = "form"
```

Comment:
there is an id, that can be used to point to the submit button, if you use that, you won't need the `submitForm` method.